scripts/lib/install_node.sh

```diff
@@ -0,0 +1,21 @@
+# Install Node and necessary frameworks/packages
+
+# Install Node.js on Arch Linux
+install_node() {
+  command -v node >/dev/null 2>&1 || {
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+    source ~/.nvm/nvm.sh
+    nvm install 12.15.0
+  };
+}
+
+# Install dependencies for notification folder
+init_notif() {
+  cd ../src/notifications && npm install 
```

Comment:
Add cd - to return to previous directory. I.e. ensure that the current working directory is the same as when the install function is called.